pkg/agent/route/route_linux.go

```diff
@@ -891,6 +891,12 @@ func (c *Client) Reconcile(podCIDRs []string) error {
 		if desiredPodCIDRs.Has(route.Dst.String()) {
 			continue
 		}
+		// A route destined to an IPv6 link-local CIDR is always system auto-generated along with a link-local
+		// address, which is not configured by antrea and is supposed to be ignored in the "deletion" list.
```

Comment:
I think we should also add the route to https://github.com/antrea-io/antrea/blob/99a5f2522a1460037b8b7a558838d603ed9d6e77/pkg/agent/route/route_linux.go#L280-L285 given we already have logic to recover required routes and to make bumping up release can fix broken clusters. Otherwise the cluster would not recover until Nodes are rebooted.

Reply:
updated.